### PR TITLE
Add specific practices on Kobiton Automation testing

### PR DESCRIPTION
**Pull request by taynguyen**
@duylam : Please take a look.

### CODE REVIEW

File: java/src/test/java/com/kobiton/usecases/practices/RetrieveAutomationDetails.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        capabilities.setCapability("deviceName", "Galaxy J7");
+        capabilities.setCapability("platformName", "Android");
+        
+        AndroidDriver<WebElement> driver = new AndroidDriver<>(getAutomationUrl(), capabilities);
+        
+        try {
+            driver.manage().timeouts().implicitlyWait(60, TimeUnit.SECONDS);
+            driver.findElementByXPath("//*[@resource-id='android:id/home']").click();
+        } catch (Exception e) {
+            driver.quit();
```
Comment (khanhdodang) [inline java/src/test/java/com/kobiton/usecases/practices/RetrieveAutomationDetails.java]: should be
```
if (driver != null) {
  driver.quit();
}
```

File: java/src/test/java/com/kobiton/usecases/practices/RetrieveAutomationDetails.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        Response response = client.newCall(request).execute();
+        Assert.assertNotNull(response.body(), "Response body is null");
+        
+        // responseBodyString is JSON format, you could parse to use it as your demand
+        String responseBodyString = response.body().string();
+        System.err.println("response body: " + responseBodyString);
+        Assert.assertNotNull(responseBodyString, "Response body string is null");
+    }
+
+    private AndroidDriver<WebElement> executeSimpleAppTest() throws MalformedURLException {
```
Comment (khanhdodang) [inline java/src/test/java/com/kobiton/usecases/practices/RetrieveAutomationDetails.java]: why do we have a test here?

Comment (taynguyen) [reply java/src/test/java/com/kobiton/usecases/practices/RetrieveAutomationDetails.java]: In this case, we execute a automation test, then use api to get detailed result.

File: java/src/test/java/com/kobiton/usecases/practices/RetrieveAutomationDetails.java (hunk 1 above)
Comment (taynguyen) [reply java/src/test/java/com/kobiton/usecases/practices/RetrieveAutomationDetails.java]: Not necessary. Driver couldn't null here.

**Pull request merged**